common4j/src/main/com/microsoft/identity/common/java/jwt/AbstractJwtRequest.java

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.common.java.jwt;
+
+import com.google.gson.annotations.SerializedName;
+import com.microsoft.identity.common.java.util.StringUtil;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+@SuppressFBWarnings({"URF_UNREAD_FIELD", "URF_UNREAD_FIELD"})
+public abstract class AbstractJwtRequest {
+
+    @Setter
+    @Accessors(prefix = "m")
+    @SerializedName("ctx")
+    @SuppressFBWarnings(value = "URF_UNREAD_FIELD", justification = "This is output through serialization")
```

Comment:
You are probably getting these because you don't have a getter. Do you really not need a getter on these at all?

Reply:
It has been in use only for serialization, I guess that's why no getters. I will probably add them (separate change) to aid in UT.